From: Lucas Kanashiro <kanashiro@debian.org>
Date: Fri, 18 Dec 2020 20:41:56 -0300
Subject: Disable pry

Forwarded: not-needed
Last-Update: 2017-09-01
---
 spec/spec_helper.rb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/spec/spec_helper.rb b/spec/spec_helper.rb
index 1ae890b..9108b30 100644
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -2,7 +2,8 @@ require 'simplecov'
 require 'webmock'
 require 'vcr'
 
-require 'pry' if RUBY_VERSION > "1.8.7"
+# We do not need pry to run tests during the build
+#require 'pry' if RUBY_VERSION > "1.8.7"
 
 class InceptionFormatter
   def format(result)
